app/controllers/tickets_controller.rb

```diff
@@ -154,7 +156,7 @@ def details_before_anonymization
 
     render json: {
       user: user,
-      person: person,
+      person: person&.serializable_hash(private_attributes: person_private_attributes),
```

Comment:
```suggestion
      person: person&.as_json(private_attributes: person_private_attributes),
```

Reply:
Not a big difference, but more Rails-esque

Reply:
Okay changed.